### ISSUE TITLE
Laravel 9 getKey method not implemented

### ISSUE THREAD

**Issue by cursosdesarrolloweb** [stechstudio/laravel-env-security#21]
Hi,

Laravel 9 Encrypter contract need implements the getKey method.](https://laravel.com/docs/9.x/upgrade)

The Illuminate\Contracts\Encryption\Encrypter contract now defines a getKey method. If you are manually implementing this interface, you should update your implementation.

Error: Class STS\EnvSecurity\Drivers\KmsDriver contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (Illuminate\Contracts\Encryption\Encrypter::getKey).

**Comment by n0Sp00n** [comment]
I confirm this is still an issue.

Let us know how to resolve that because a Production Application is broken at the moment because of that.

**Comment by cursosdesarrolloweb** [comment]
@n0Sp00n It seems that the best option at the moment is fork this repository and update it, looks abandoned.

**Comment by n0Sp00n** [comment]
This should do the work, right?

```
public function getKey() {

        $key = env('APP_KEY');
        if (Str::startsWith($key, 'base64:')) {
           $key = base64_decode(substr($key, 7));
        }

        return $key;
}
```

**Comment by cursosdesarrolloweb** [comment]
Not sure, it works for me:

```php
public function getKey() {
    return $this->keyId;
}
```

**Comment by jszobody** [comment]
This package isn't abandoned at all. I'm just not using it in Laravel 9 myself yet and haven't hit this.

I'd welcome a PR, if any of y'all want to submit one. That would be the fastest way to get this resolved. Otherwise it'll be a week or two until I can handle.

**Comment by n0Sp00n** [comment]
> ERROR: Permission to stechstudio/laravel-env-security.git denied to n0Sp00n.
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.
> 

Could you please assist me resolving this issue? @jszobody

**Comment by jszobody** [comment]
Hi @n0Sp00n. Are you trying to push a change directly to this repository? You won't have permission to do that. You need to submit a Pull Request.

**Comment by jszobody** [comment]
Are we thinking the `getKey()` needs to actually return anything? 

Both current drivers (AWS KMS, and Google KMS) don't have local keys, there's nothing to return. Returning the `APP_KEY` as @n0Sp00n suggested doesn't make sense to me, as it doesn't have anything to do with what this package is doing.

Sounds like we need a stub method just to comply with the interface, returning an empty string. Anyone disagree?

**Comment by n0Sp00n** [comment]
I agree with you. Later on I realized App_key has nothing to do with it.

**Comment by jszobody** [comment]
This is now fixed, upgrade to version 1.14.

**Comment by n0Sp00n** [comment]
Thanks a lot.

Cheers!
